securedrop/tests/test_manage.py

```diff
@@ -14,6 +14,10 @@ def test_parse_args(self):
         # just test that the arg parser is stable
         manage.get_args()
 
+    def test_setup_verbose(self):
+        args = manage.get_args().parse_args(['run'])
+        manage.setup_verbosity(args)
```

Comment:
There is no such thing as restoring coverage without adding tests. It is acceptable to explicitly mark code as not covered by tests (e.g. [`# pragma: no cover`](https://coverage.readthedocs.io/en/latest/excluding.html)), if you do not think it is necessary or worthwhile to test certain sections of code; however, it is not acceptable to trick the coverage analyzer into thinking new code is covered without actually testing the new code, which is what you did here.